apps/src/code-studio/components/ShareAllowedDialog.jsx

```diff
@@ -10,7 +10,7 @@ import AbuseError from './AbuseError';
 import SendToPhone from './SendToPhone';
 import color from '../../util/color';
 import * as applabConstants from '../../applab/constants';
-import * as gamelabConstants from '@cdo/apps/p5lab/constants';
+import * as p5labConstants from '@cdo/apps/p5lab/constants';
```

Comment:
Since we're only using GAME_HEIGHT and GAME_WIDTH, would it make sense to just import those two?
i.e. `import {GAME_HEIGHT, GAME_WIDTH} from '@cdo/apps/p5lab/constants';`

Reply:
Renamed to `APP_WIDTH` and `APP_HEIGHT`, and since applabConstants contains those too, leaving the imports as is so we can differentiate.